### PR TITLE
Add wasm ibc example test into v3 branch

### LINKED ISSUE #15: Add tests for wasm
example contract for sending cw20 across IBC https://github.com/CosmWasm/cw-plus/tree/main/contracts/cw20-ics20
example reflect test https://github.com/notional-labs/wasmd/blob/bounty/x/wasm/ibc_reflect_test.go

### PR DESCRIPTION
**Pull request by misko9**
Resolves #15

### CODE REVIEW

File: wasm/wasm.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package wasm
+
+import (
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	"github.com/strangelove-ventures/ibctest/v3/chain/cosmos"
+	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
+)
+
+func WasmEncoding() *simappparams.EncodingConfig {
```
Comment (agouin) [inline wasm/wasm.go:9]: Since this is cosmos chain specific, might make sense for this wasm package to be chain/cosmos/wasm

Comment (misko9) [reply wasm/wasm.go:9]: Yeah, that makes sense, I update that.

**Review by agouin — APPROVED** [review]
smart contracts in ibctest, so fancy :tada:

**Comment by jackzampolin** [comment]
This is real fancy! LFG

**Review by jackzampolin — APPROVED** [review]

**Pull request merged**